build/config.props

```diff
@@ -14,7 +14,7 @@
     <!-- when changing any of the NuGetVersion props below, update ProductVersion in src\NuGet.Clients\NuGet.Tools\NuGetPackage.cs and run tools-local\ship-public-apis -->
     <MajorNuGetVersion Condition="'$(MajorNuGetVersion)' == ''">5</MajorNuGetVersion>
     <MinorNuGetVersion Condition="'$(MinorNuGetVersion)' == ''">11</MinorNuGetVersion>
-    <PatchNuGetVersion Condition="'$(PatchNuGetVersion)' == ''">3</PatchNuGetVersion>
+    <PatchNuGetVersion Condition="'$(PatchNuGetVersion)' == ''">4</PatchNuGetVersion>
```

Comment:
please see the comment just above

Reply:
Great catch Andy. Fixed in the latest commit. Thank you.